certbot/constants.py

```diff
@@ -32,6 +32,7 @@
     auth_cert_path="./cert.pem",
     auth_chain_path="./chain.pem",
     strict_permissions=False,
+    reuse_key=False,
```

Comment:
For the record, though I don't think anyone has proposed this seriously, I would vote to reject any change of reuse becoming the default.